lib/services/hpcpackage_remain.pm

```diff
@@ -89,10 +91,14 @@ sub full_pkgcompare_check {
     my $stage = $hash{stage};
 
     if ($stage eq 'before') {
+        # It need python2 module to compare packages
```

Comment:
Add the poo in the comments will be better.

Reply:
After long time later, the poo cannot be opened. It will useless add those at here.  I can add POO at description of PR

Reply:
We won't delete any poo why can't open?